test/featuregates_linter/test_data/versioned_feature_list.yaml

```diff
@@ -9,7 +9,7 @@
     preRelease: Beta
     version: "1.28"
   - default: true
-    lockToDefault: false
+    lockToDefault: true
```

Comment:
you're updating these via `./hack/update-featuregates.sh` right?

Reply:
Yes